android/filament-android/src/main/java/com/google/android/filament/Engine.java

```diff
@@ -300,9 +300,9 @@ public void finalize() {
     public static class Config {
 
         // #defines in Engine.h
-        private static final long FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB = 3;
-        private static final long FILAMENT_PER_FRAME_COMMANDS_SIZE_IN_MB = 2;
-        private static final long FILAMENT_MIN_COMMAND_BUFFERS_SIZE_IN_MB = 1;
+        private static final long FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB = 9;
```

Comment:
Should we write a comment in any of these files informing about the crash in larger models?

Reply:
There're ppl facing similar issue & filament guys just recommend to increase those constants to their needs.  I will leave a note in our project notes folder